bin/supplies.js

```diff
@@ -55,6 +45,30 @@ program
       driver.secret = options.key;
     }
 
+    // eslint-disable-next-line max-len
+    if (driver.supports.blockchains.length > 1 && !driver.supports.blockchains.includes(options.blockchain)) {
```

Comment:
This way there is no need for disabling the Eslint rule 😄 
```suggestion
    if (
      driver.supports.blockchains.length > 1
      && !driver.supports.blockchains.includes(options.blockchain)
    ) {
```